src/libraries/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/UnconditinalSuppressMessageAttribute.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#if NETSTANDARD2_0
+#nullable enable
+
+namespace System.Diagnostics.CodeAnalysis
```

Comment:
Rather than adding another copy of this file, could you use a `<Compile Include="..."` to include the existing file from the corelib project?